docs/tables/aws_backup_framework.md

```diff
@@ -0,0 +1,169 @@
+
+```sql
+select
+  framework_name
+from
+  aws_backup_framework,
+  jsonb_array_elements(framework_controls) as controls
+where
+  controls ->> 'ControlName' = 'BACKUP_RESOURCES_PROTECTED_BY_BACKUP_VAULT_LOCK';
+```
+
+### List control name and control scope for backup AWS Backup Frameworks
+
+This query will return an empty control scope if the control doesn't apply to a specific AWS resource type.
+Otherwise if query will list the control name and the AWS resource type.
+
+```sql
+select
+  framework_name,
+  controls ->> 'ControlName' as control_name,
+  jsonb_array_elements_text(
+    case when (controls -> 'ControlScope' ->> 'ComplianceResourceTypes') is null then to_jsonb('[""]'::json)
+    else (controls -> 'ControlScope' -> 'ComplianceResourceTypes')
+    end
+  ) as control_scope
+from
+  aws_backup_framework,
+  jsonb_array_elements(framework_controls) as controls
+```
+
+### List control name and control scope for a specific AWS Backup Framework
+
+This query will return an empty control scope if the control doesn't apply to a specific AWS resource type.
+Otherwise if query will list the control name and the AWS resource type.
+
+```sql
+select
+  framework_name,
+  controls ->> 'ControlName' as control_name,
+  jsonb_array_elements_text(
+    case when (controls -> 'ControlScope' ->> 'ComplianceResourceTypes') is null then to_jsonb('[""]'::json)
+    else (controls -> 'ControlScope' -> 'ComplianceResourceTypes')
+    end
+  ) as control_scope
+from
+  aws_backup_framework,
+  jsonb_array_elements(framework_controls) as controls
+where
+  framework_name = 'framework_name'
+```
+
+### Query to create the AWS config rule name from the AWS Backup Framework controls
+
+The below query will demonstrate how to create the AWS Config rule name from the AWS Backup Framework control.
+If the rule exists in AWS Config, then the end user can query the AWS Config table to see if resources in the account are compliant to the rule and what remediation action should take place.
+
+```sql
+select
+  case when framework_information.control_scope = '' then concat(framework_information.control_name, '-', framework_information.framework_uuid)
+  else concat(upper(framework_information.control_scope), '-', framework_information.control_name, '-', framework_information.framework_uuid)
+  end as rule_name
+from
+(
+  select
+    framework_name,
+    controls ->> 'ControlName' as control_name,
+    jsonb_array_elements_text(
+      case when (controls -> 'ControlScope' ->> 'ComplianceResourceTypes') is NULL THEN to_jsonb('[""]'::json)
+      else (controls -> 'ControlScope' -> 'ComplianceResourceTypes')
+      end
+    ) as control_scope,
+    right(arn, 36) as framework_uuid
+  from
+    aws_backup_framework,
+    jsonb_array_elements(framework_controls) as controls
+) as framework_information;
+```
+
+### Querying rules in AWS Config from AWS Backup Framework rules
```

Comment:
Does the Backup API return any information on a framework's compliance status or the count on compliant/non-compliant controls (which is available in the Summary page per https://docs.aws.amazon.com/aws-backup/latest/devguide/viewing-frameworks.html), or do we always need to join with the Config table(s) to get this info?